docs/sources/setup-grafana/installation/debian/index.md

```diff
@@ -11,17 +11,24 @@ weight: 100
 
 This page explains how to install Grafana dependencies, download and install Grafana, get the service up and running on your Debian or Ubuntu system, and also describes the installation package details.
 
-## Repository migration (November 8th 2022)
+## Note on upgrading
 
-From that date, Grafana packages will be served from a new repository (<packages.grafana.com/deb/{product}> -> <apt.grafana.com>). The new repository serves, from a single APT configuration, all Grafana OSS products, as well as Grafana Enterprise.
+While the process for upgrading Grafana is very similar to installing Grafana, there are some key backup steps you should perform. Read [Upgrading Grafana]({{< relref "../../../upgrade-guide/" >}}) for tips and guidance on updating an existing installation.
 
-The old URLs will still work, serving the content from the new repository, but you may encounter warnings about some repository attributes changing (e.g. `Origin` and `Label`).
+## Download and install Grafana
 
-## Note on upgrading
+Complete the following steps to download and install Grafana:
 
-While the process for upgrading Grafana is very similar to installing Grafana, there are some key backup steps you should perform. Read [Upgrading Grafana]({{< relref "../../../upgrade-guide/" >}}) for tips and guidance on updating an existing installation.
+1. On the [Grafana download page](https://grafana.com/grafana/download), select the Grafana version you want to install.
+   - The most recent Grafana version is selected by default.
+   - The **Version** field displays only finished releases. If you want to install a beta version, click **Nightly Builds** and then select a version.
```

Comment:
Beta releases are still tagged and will show up in the version select dropdown (though the most recent stable release will still be the default), the user only needs to click the Nightly Builds button if they want to run a nightly build (though I don't think many people actually do that, will dig into the usage reports).
```suggestion
   - The **Version** field displays only tagged releases. If you want to install a nightly build, click **Nightly Builds** and then select a version.
```